lstchain/scripts/lstchain_create_run_summary.py

```diff
@@ -296,10 +371,17 @@ def main():
 
     reference_counters = [read_counters(date_path, run) for run in run_numbers]
 
-    run_types = [
-        type_of_run(date_path, run, counters)
-        for run, counters in zip(run_numbers, reference_counters)
-    ]
+    if is_db_available("lst101-int"):
```

Comment:
I would probably add tcu_server (set by default as `'lst101-int'` ) as input argument to make it more flexible and change it accordingly across the script. Otherwise it looks good to me